### ISSUE TITLE
HACKING.md should be part of the main documentation

### ISSUE THREAD

**Issue by lemire** [simdjson/simdjson#801]
It is really more like "simdjson for contributors".

**Comment by jkeiser** [comment]
Yeah. Actually just linking to it from the documentation list is probably the right thing. I noticed the other day it's buried way at the bottom, and it shouldn't be.

**Comment by lemire** [comment]
The README now links `HACKING.md` from its "Contributing to simdjson" section, next to `CONTRIBUTING.md`, as the guide to the source layout, building and design. That was the intent here, so I'm closing this.
